test/setup-globals.js

```diff
@@ -0,0 +1,15 @@
+const fetch = require("node-fetch");
+
+// Expose a global fetch polyfill
+global.fetch = fetch;
+global.Headers = fetch.Headers;
+
+// jsdom FormData implementation is inconsistent, exposing a better one
+global.FormData = require("form-data");
+
+// atob polyfill for tests
+global.atob = require("atob");
+
+global.Blob = function Blob(dataArray) {
+  return Buffer.from(dataArray[0]);
+};
```

Comment:
Can't we take advantage of what's in `blob.js` here?

Reply:
Turns out we can! Mocha will load the setup file even if it's TypeScript, so I've updated it to import from `blob.js`.